libkineto/src/CuptiActivityInterface.cpp

```diff
@@ -92,6 +92,35 @@ int CuptiActivityInterface::smCount() {
   return sm_count;
 }
 
+#ifdef HAS_CUPTI
+static std::vector<cudaOccDeviceProp> getOccDeviceProp() {
```

Comment:
Since this is not CUPTI let's move it into its own file, as suggested below... e.g. CudaDeviceProperties.h/cpp

Reply:
Thanks. Removed to CudaDeviceProperties.cpp